src/main/java/Percy.java

```diff
@@ -0,0 +1,82 @@
+import java.io.IOException;
+
+import percy.command.Command;
+import percy.command.Parser;
+import percy.exception.PercyException;
+import percy.storage.Storage;
+import percy.task.TaskList;
+import percy.ui.Ui;
+
+public class Percy {
+    private TaskList tasks;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * Constructs the chat bot Percy.
+     */
+    public Percy() {
+        this.ui = new Ui();
+        this.storage = new Storage();
+        this.tasks = new TaskList(storage.load());
+    }
+
+    /**
+     * Runs the chat bot which accepts commands and replies accordingly.
+     *
+     * @throws IOException
+     */
+    /*
```

Comment:
Not sure if you should still keep this commented-out code here since it seems to have already been replaced with an updated version below?